templates/kubernetes/documentation.html

```diff
@@ -0,0 +1,165 @@
+{% extends "kubernetes/base_kubernetes.html" %}
+
+{% block title %}Secure, enterprise-grade Kubernetes for multi-cloud operations{% endblock %}
+
+{% block meta_description %}
+  Canonical's securely designed enterprise-grade Kubernetes builds up from Ubuntu OS to hybrid multi-cloud container orchestration clouds, edge and IoT, managed Kubernetes and enterprise support packages.
+{% endblock meta_description %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1b018GoKMF3abEFJlUkfAEK2JEhXWICKF4V0dGdTvgJs/edit
+{% endblock meta_copydoc %}
+
+{% from "_macros/vf_hero.jinja" import vf_hero %}
+{% from "_macros/vf_quote-wrapper.jinja" import vf_quote_wrapper %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-section--hero">
```

Comment:
can you use a hero macro here?

Reply:
Unfortunately not. I have raised it with DS though

Reply:
Julie got back to be, it is a bug that was fixed in a more recent vanilla. Looks likes we are a couple versions behind. I made [an issue](https://warthogs.atlassian.net/browse/WD-19622) for it